server/service/src/document/document_service.rs

```diff
@@ -0,0 +1,440 @@
+use chrono::Utc;
```

Comment:
do we need to add _service to document_service and json_schema_service ?

Reply:
found it quit helpful to identify the service files

Reply:
ok, for your info in vs code, `cmd+p` then `servicediscovery` is probably the fastest way to navigate to a file